modules/ROOT/attachments/admin-api.yaml

```diff
@@ -947,6 +947,127 @@ paths:
+  /cluster/partitions:
+    get: 
+      summary: Get cluster-level metadata for all partitions in cluster
+      description: Get cluster-level metadata for all partitions in cluster.
+      operationId: get_cluster_partitions
+      parameters: 
+        - name: disabled
+          in: query 
+          required: false
+          schema:
+            type: boolean
+        - name: with_internal
+          in: query
+          required: false
+          schema: 
+            type: boolean
+      responses:
+        200:
+          description: Partition metadata
+          content: 
+            application/json:
+              schema: 
+                type: array
+                items:
+                  $ref: '#/components/schemas/cluster_partition'
+      tags: 
+        -  Partitions    
+  /cluster/partitions/{namespace}/{topic}:
+    post:
+      summary: Disable/enable all partitions of a topic
+      description: "Disable or enable all partitions of a topic. To disable, use `disabled: true` in request body. To enable, use `disabled: false`."
+      operationId: post_cluster_partitions_topic
+      parameters: 
+        - name: namespace
+          in: path
+          required: true
+          schema: 
+            type: string
+        - name: topic
+          in: path
+          required: true
+          schema: 
+            type: string
+      requestBody: 
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/disable_enable_partitions'
+        required: false 
+      responses: 
+        200:
+          description: Partitions disabled or enabled successfully
+          content: {}
+      tags: 
+        -  Partitions 
+    get:
+      summary: Get cluster-level metadata for all partitions in topic
+      description: Get cluster-level metadata for all partitions in topic.
+      operationId: get_cluster_partitions_topic
+      parameters: 
+        - name: namespace
+          in: path
+          required: true
+          schema: 
+            type: string
+        - name: topic
+          in: path
+          required: true
+          schema: 
+            type: string
+      requestBody: 
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/disable_enable_partitions'
+        required: false  
+      responses: 
+        200:
+          description: Topic partitions metadata
+          content: 
+            application/json:
+              schema: 
+                type: array
+                items:
+                  $ref: '#/components/schemas/cluster_partition'
+      tags: 
+        -  Partitions 
+  /cluster/partitions/{namespace}/{topic}/{partition}:
+    post: 
+      summary: Disable/enable a single partition
+      description: "Disable or enable a single partition. To disable, use `disabled: true` in request body. To enable, use `disabled: false`."
```

Comment:
`disabled: true/false` should probably be `{"disabled": true/false}` to underscore that this is json